db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,35 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
+      t.references :evm_owner,     :type => :bigint, :index => true, :references => :user
+      t.references :miq_group,     :type => :bigint, :index => true
+      t.string :type, :index => true
+      t.string :name
+      t.jsonb :payload, :default => {}
+      t.jsonb :credentials, :default => {}
+
+      t.timestamps
+    end
+
+    create_table :workflow_instances do |t|
+      t.references :ems,      :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :workflow, :type => :bigint, :index => true
+      t.references :tenant,   :type => :bigint, :index => true
+      t.references :evm_owner,     :type => :bigint, :index => true, :references => :user
+      t.references :miq_group,     :type => :bigint, :index => true
+      t.references :miq_task, :type => :bigint, :index => true
+      t.string :type, :index => true
+      t.string :userid
```

Comment:
Is this different from evm_owner?

Reply:
I added evm_owner because it was in the OwnershipMixin but maybe we don't need both.  @kbrock @Fryguy what do you recommend for tracking ownership?  I'm used to seeing `userid` on things like miq_task and not evm_owner.  I also don't see `miq_group_id` used very much where tenant_id is used more.